modules/zstream-4-20-5-enhancements.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * release_notes/ocp-4-20-release-notes.adoc
+
+
+:_mod-docs-content-type: REFERENCE
+[id="zstream-4-20-5-enhancements_{context}"]
+= Enhancements
+
+[role="_abstract"]
+The following enhancements for direct authentication with an external OpenID Connect (OIDC) identity provider are included in this z-stream release:
+
+General availability of direct authentication with an external OIDC identity provider::
```

Comment:
Are DLs for z-stream updates typical? I don't spend a lot of time in RNs outside of what's in my purview, to be fair, but I worry that the semantic relationship established here doesn't make sense.